taskvine/src/manager/taskvine.h

```diff
@@ -889,6 +889,12 @@ void vine_manager_install_library( struct vine_manager *m, struct vine_task *t,
 */
 void vine_manager_remove_library( struct vine_manager *m, const char *name );
 
+/** Get the library task with the given name, return null if not found.
+@param m A manager object
+@param library_name The library name
+*/
+struct vine_task *vine_manager_get_library(struct vine_manager *m, const char *library_name);
```

Comment:
Move to `vine_manager.h`. This is not part of the official API.

Reply:
Actually you already have it in `vine_manager.h`.  Delete from here.

Reply:
I am not sure how could we invoke it from `cvine.py` if I delete it from `vine_manager.h`?

Reply:
Since I have 
``` Python
def check_library_exists(self, library_name):
    return cvine.vine_manager_get_library(self._taskvine, library_name) is not None
``` 
in `manager.py`, it seems that this API is supposed to be exposed?

Reply:
It is exposed in the bindings when you add it to `vine_manager.h`. `vine_manager.h` declares functions related to the manager that are not exposed in the API.  API functions go to `taskvine.h`. This declaration should be in `vine_manager.h` and not `taskvine.h`.  Look at `taskvine.i` somewhere in the bindings source directory to see which `.h` files are included to construct the bindings.

Reply:
It seems that we didn't include `vine_manager.h` to construct bindings, I added it so that we can invoke in python. Does that look good?

Reply:
Good catch!